cmd/system-probe/config/adjust_npm.go

```diff
@@ -99,4 +101,26 @@ func adjustNetwork(cfg config.Config) {
 		log.Warn("disabling NPM connection rollups since USM connection rollups are not enabled")
 		cfg.Set(netNS("enable_connection_rollup"), false, model.SourceAgentRuntime)
 	}
+
+	// disable features that are not supported on certain
+	// configs/platforms
+	var disableConfigs []struct {
+		key, reason string
+	}
+	if ebpflessEnabled {
+		const notSupportedEbpfless = "not supported when ebpf-less is enabled"
+		disableConfigs = append(disableConfigs, []struct{ key, reason string }{
+			{spNS("enable_conntrack_all_namespaces"), notSupportedEbpfless},
```

Comment:
Does ebpf-less imply root-less as well? Doesn't the netlink conntracker support all namespaces?